docs/usage/semantic-commits.md

```diff
@@ -15,7 +15,8 @@ When Renovate finds Angular-style commits, Renovate will create commit messages
 - chore(deps): update eslint to v4.2.0
 - fix(deps): update express to v4.16.2
 
-Renovate uses `chore` by default, but uses `fix` for updates to your production dependencies in your `package.json` file.
+Renovate uses the `chore` prefix by default.
+If you extend `config:base`, Renovate still defaults to `chore`, but will use the `fix` prefix for dependencies.
```

Comment:
I think the fix will only applied to npm package dependencies. 🤔

Reply:
npm and composer I believe

Reply:
And by `dependencies` we mean production dependencies and not `devDependencies`

Reply:
```suggestion
When you extend `config:base`, Renovate still defaults to `chore`, but will use the `fix` prefix for npm production dependencies (`devDependencies` still use `chore`).
```

How about this sentence?